templates/display.tpl

```diff
@@ -34,7 +34,7 @@
 		</a>
 
 		<a href="{$parentUrl}" class="title">
-			{$title|escape}
+			{$title}
```

Comment:
This will work for article titles, but I think this won't properly escape issue galley titles. Those should be treated as plain-text. For example, an issue galley with a title entered as:

> Beyond the P factor: p < 0.05

...should have its title escaped in the markup as:

```html
Beyond the P factor: p &lt; 0.05
```

For an article galley with the same title, it will already be stored in the database as HTML, and won't need further escaping.